src/cmd1.cc

```diff
@@ -556,89 +554,6 @@ s16b tot_dam_aux(object_type *o_ptr, int tdam, monster_type *m_ptr,
 
 
 /*
- * Search for hidden things
- */
-void search(void)
```

Comment:
Yeah, I think just making them visible would be OK. Not quite as thematic as hidden doors, but I suppose it'll do... but OTOH how much of the deleted code would really have to be kept to support hidden doors? AFAICT it's not actually that much.

(I think I would be in favor of keeping them, letting Sense Hidden find them automatically and just giving every class a sort of base-line chance of finding them every turn, e.g. by just pretending everyone has, say, 10 points in Sneakiness and hard-coding the effect that 10 Sneakiness would have... if that makes sense.)